### PR TITLE
Add infrastructure nodes sizing recommendations for 4.7

### PR DESCRIPTION
**Pull request by mohit-sheth**
This commit adds information on infrastructure nodes which hosts
Router, Registry, Logging and Monitoring components for various
nodes scale. The data is based on the cluster maximums and density
focused scale test runs.

### CODE REVIEW

File: modules/infrastructure-node-sizing.adoc [hunk 1]
```diff
@@ -7,44 +7,33 @@
...
-| 32
-| 192
 
 |===
 
 [IMPORTANT]
 ====
 These sizing recommendations are based on scale tests, which create a large number of objects across the cluster. These tests include reaching some of the cluster maximums. In the case of 250 and 500 node counts on a {product-title} {product-version} cluster, these maximums are 10000 namespaces with 61000 pods, 10000 deployments, 181000 secrets, 400 config maps, and so on. Prometheus is a highly memory intensive application; the resource usage depends on various factors including the number of nodes, objects, the Prometheus metrics scraping interval, metrics or time series, and the age of the cluster. The disk size also depends on the retention period. You must take these factors into consideration and size them accordingly.
+
+The sizing recommendations are applicable only for the infrastructure components which gets installed during the cluster install - Prometheus, Router and Registry. Logging is a day two operation and the recommendations do not take it into account.
```
Comment (ahardin-rh) [inline modules/infrastructure-node-sizing.adoc:36]: These sizing recommendations are only applicable for the Prometheus, Router, and Registry infrastructure components, which are installed during cluster installation. Logging is a day-two operation and is not included in these recommendations.

**Comment by ahardin-rh** [comment]
@mffiedler PTAL

**Review by mffiedler — APPROVED** [review]
LGTM

File: modules/infrastructure-node-sizing.adoc (hunk 1 above)
Comment (mohit-sheth) [reply modules/infrastructure-node-sizing.adoc]: done, thanks

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Review by jeana-redhat — APPROVED** [review]
approving exception

**Pull request merged**